site/content/en/docs/contributing/development-environment.md

```diff
@@ -73,25 +80,36 @@ description: 'Installing a development environment for different operating syste
   >
   > Read this article [Node Sass does not yet support your current environment](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
 
-- Open new terminal (Ctrl + Shift + T), run Visual Studio Code from the virtual environment
+- Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/) and [Docker-Compose](https://docs.docker.com/compose/install/)
```

Comment:
Or docker-desktop for Mac & Windows

Reply:
But below, in Note for windows users, it says to install WSL and then follow all the commands for Ubuntu... So why would we need to install Docker for Windows?
For Mac: Maybe it is better to change the link for [this](https://docs.docker.com/engine/install/).